src/results/result-form.resource.ts

```diff
@@ -343,13 +343,43 @@ export async function UpdateEncounter(uuid: string, payload: any) {
 
 //TODO: the calls to update order and observations for results should be transactional to allow for rollback
 export async function UpdateOrderResult(
-  encounterUuid: string,
   obsPayload: any,
   orderPayload: any,
   encounterPostData: any
 ) {
   const abortController = new AbortController();
 
+  const patientVisits = await openmrsFetch(
```

Comment:
Instead of fetching all visits and looping through them, it is possible to use the order's encounter and fetch the associated visit?

Reply:
I couldn't figure out a way to do that unfortunately.

Reply:
@CynthiaKamau  @arodidev  could you be of assistance to @reagan-meant

Reply:
@reagan-meant It should just be: `encounters/${encounter.uuid}?v=custom:(visit:(uuid))`

Reply:
> I couldn't figure out a way to do that unfortunately.

Perhaps you can try exploring that through custom representation 

Also kindly address the conflicts